docs-chef-io/content/inspec/profiles.md

```diff
@@ -170,6 +187,7 @@ supports:
   - platform: aws
 ```
 
+
```

Comment:
```suggestion
```

Reply:
A single blank line is needed between the yaml code and the header. Line 189 fulfills this requirement already.